tests/Integration/tests/codehighlight-languages/input/index.rst

```diff
@@ -0,0 +1,7 @@
+===============================
+Registered code-block languages
+===============================
+
+All registered languages for code blocks:
+
+..  codeblock-languages::
```

Comment:
I would suggest that we use a common domain for directives that are used to document our rst-implementation. Maybe 

rest:codeblock-languages

Ist seems like we are gonna have more then just one

Reply:
This is valid?

    .. rest:codeblock-languages::

How do I name then the according template respective the path?

Reply:
just like we do in the php domain?

Reply:
directive/rest/codeblock-languages.html.twig

Reply:
Why "rest"? Maybe "t3" would be better as it is more an internal directive?

Reply:
Or "internal"? That's make the purpose clear.

Reply:
h2document?

Reply:
OMG, is "h2document" = "how to document"? This is so not clear :-P

I would vote for "guides", "t3", "internal" in that order...

Reply:
I like guides. I would reserve t3 for typo3 specific stuff, like if we ever want to do something expecially for TypoScript or whatever?

Reply:
Okay, will use guides :1st_place_medal:

Reply:
Done, please have a look.